docs/getting_started/basics/execute.md

```diff
@@ -53,3 +55,43 @@ The "Live Execution" action allows to trigger the execution of the whole workflo
       <figcaption><b>Figure 5</b>: Executing workflow in live execution mode.</figcaption>
 </figure>
 
+## Threaded Execution
+
+By default Shift will run the execution of the workflow on a dedicated thread. This allows the user to freely inspect the board that is currently executing. Additionally, for long executing workflows, the user is free to author and inspect any board other than the executing one.
+
+### Stopping a Running Execution
+
+An interruption of the currently executing workflow can be requested by clicking on the *Clear Execution* button. While a board is executing, this button will force the execution to stop without clearing the execution progress.
+
+<figure>
+      <img src="images/stopping_execution.gif" alt="Stopping the Execution">
+      <figcaption><b>Figure 6</b>: Stopping the execution of a workflow.</figcaption>
+</figure>
+
+The interruption will take effect once the execution of the operator that was running when the button was clicked ends. This behaviour ensures that Shift can safely kill the execution thread.
+
+### Managing Threaded Execution Behaviour
+
+The threaded execution feature can be activated or deactivated from the *Single Thread* toggle in *Execute* menu in the top bar.
```

Comment:
```suggestion
The threaded execution feature can be activated or deactivated from the *Single Thread* toggle in *Execute* menu in the top bar. Please be aware that disabling this feature will cause the workflow to run in the same Python process as the Shift UI, resulting in Shift freezing until the execution is complete.
```

Reply:
Changed "process" to "thread", let me know if it is alright and I can push

```suggestion
The threaded execution feature can be activated or deactivated from the *Single Thread* toggle in *Execute* menu in the top bar. Please be aware that disabling this feature will cause the workflow to run in the same Python thread as the Shift UI, resulting in Shift freezing until the execution is complete.
```

Reply:
It's good